samples/react/src/components/ColorSwatches/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ColorSwatches = ({ colors }) => (
+    <section className="sg-color-swatches">
+        {colors.map((color, index) => <Row key={index} {...color} />)}
+    </section >
+);
+
+const Row = ({ rowName, swatches }) => (
```

Comment:
Nice co-habitation of components in this file.

Reply:
They're living in sin.